build.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pybuilder.core import use_plugin, init, Author
+
+authors = [Author("Debasish Kanhar", "dekanhar@in.ibm.com")]
+description = "Python client drivers for JanusGraph"
+copyright = "Copyright 2018 JanusGraph Python Authors"
+license = "Apache License v2.0"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'license'](https://app.codacy.com/manual/JanusGraph/janusgraph-python/pullRequest?prid=4938086)